api/apps/api/src/modules/access-control/scenarios-acl/scenario-acl.service.ts

```diff
@@ -104,4 +105,21 @@ export class ScenarioAclService implements ScenarioAccessControl {
     });
     return otherOwnersInScenario >= 1;
   }
+
+  async revokeAccess(
+    scenarioId: string,
+    userId: string,
+    loggedUserId: string,
+  ): Promise<Either<Permit, void>> {
+    if (!(await this.isOwner(loggedUserId, scenarioId))) {
+      return left(false);
+    }
+
+    if (!(await this.hasOtherOwner(userId, scenarioId))) {
+      return left(false);
```

Comment:
IMO it would be important to differentiate between the two scenarios when "Forbidden" may be returned. Specifically, I think we should return a meaningful message to the user in case they are the sole owner of a project.

Reply:
Differentiated now (and changed slightly how it is handled).

Note -> The method used needs a refactor into a common utils function because the same structure is used (and will be used) plenty of times for different entities of the ACL module.